public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/query.php

```diff
@@ -38,3 +39,24 @@ function( $var ) {
 
 	$query_ref->query_vars['type__not_in'] = $type_vars['type__not_in'];
 }
+
+/**
+ * Include custom statuses when 'all' is the status query argument.
+ *
+ * @param array            $clauses
+ * @param WP_Comment_Query $query
+ *
+ * @return string|string[]
+ */
+function comments_clauses( $clauses, $query ) {
+	$status = $query->query_vars['status'];
+	$status = (array) $status; // $query is passed by reference, don't want to alter it.
+
+	if ( in_array( 'all', $status, true ) || in_array( '', $status, true ) ) {
+		$search  = "( comment_approved = '0' OR comment_approved = '1' )";
+		$replace = "( comment_approved = '0' OR comment_approved = '1' OR comment_approved = 'inappropriate' )";
+		$clauses = str_replace( $search, $replace, $clauses );
+	}
```

Comment:
I don't think we want to show inappropriate in the "all" status— IMO it should behave like spam, and disappear, only appearing in the Inappropriate tab.

Reply:
Yeah, I think you're right.

53853129